python/paddle/tensor/math.py

```diff
@@ -1674,3 +1673,73 @@ def kron(x, y, out=None, name=None):
         check_variable_and_dtype(out, 'out', ['float16', 'float32', 'float64', 'int32', 'int64'], 'kron')
     helper.append_op(type="kron", inputs={"X": x, "Y": y}, outputs={"Out": out})
     return out
+
+
+def cumsum(x, axis=None, dtype=None, name=None):
+    """
+	:alias_main: paddle.cumsum
+	:alias: paddle.cumsum,paddle.tensor.cumsum,paddle.tensor.math.cumsum
+
+    The cumulative sum of the elements along a given axis. The first element of the result is the same of the first element of the input. 
+
+    Args:
+        x (Variable): Input of cumsum operator, the Tensor/LoDTensor needed to be cumsumed. 
+        axis (int, optional): The dimension to accumulate along. -1 means the last dimension. The default (None) is to compute the cumsum over the flattened array.
+        dtype (str|np.dtype|core.VarDesc.VarType, optional): The desired data type of returned tensor. If specified, the input tensor is casted to dtype before the operation is performed. This is useful for preventing data type overflows. The default value is None. 
+        name(str, optional): Normally there is no need for user to set this property.  For more information, please refer to :ref:`api_guide_Name`. The default value is None.  
+
+    Returns:
+        Variable(Tensor/LoDTensor): The result of cumsum operator, output of cumsum operator. 
```

Comment:
Variable->Tensor

Reply:
Tensor, the result of cumsum operator, output of cumsum operator.

Reply:
done